src/react/reducers/s3.ts

```diff
@@ -175,6 +175,24 @@ const parseExpirationDate = (expiresOn: string | undefined): Date | null => {
   return null;
 };
 
+const parseRestoreOngoingRequest = (
+  restore: string | undefined,
+): boolean | null => {
+  if (!restore) {
+    return null;
+  }
+  const extractOngoingRequest = new RegExp(/ongoing-request="([^"]+)"/);
+  const regExpExecArray = extractOngoingRequest.exec(restore);
+  if (regExpExecArray && regExpExecArray.length > 1) {
+    if (regExpExecArray[1] === 'false') {
+      return false;
+    } else if (regExpExecArray[1] === 'true') {
+      return true;
+    }
+  }
+  return null;
+};
```

Comment:
nit: I'm not sure to see the value of differencing the output of the function when `ongoing-request ` is defined or not, hence I would just make it return a boolean and return `false` instead of `null`